src/controllers/filter_controller.ts

```diff
@@ -55,7 +53,7 @@ export default class extends Controller {
         <label>
           <input type="checkbox" value="${type}" ${unselectedTypes.has(type) ? "" : "checked"}
             data-target="filter.typeInput"
-            data-action="click->filter#checkExclusively filter#render">
+            data-action="click->filter#checkExclusively change->filter#render">
```

Comment:
Adding `change->` doesn't technically, um, change anything since `change` is the [default event for input elements](https://stimulusjs.org/reference/actions#event-shorthand) in Stimulus, but it adds clarity so 👍.